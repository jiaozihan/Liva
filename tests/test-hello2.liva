class test {
    void main(){
        int a ;
        int b ;
        a=1;
        b=3;
        print ("multiple ", "params!", "\n", a, "\n" ,b, "\n");
    } 
}