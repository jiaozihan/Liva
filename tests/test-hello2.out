multiple params!
1
3
